docs/index.asciidoc

```diff
@@ -138,6 +138,7 @@ See <<plugins-{type}s-{plugin}-object_order>>.
 returns multiple columns, the data is stored as a JSON object within the field.
 <5> Takes data from the JSON object and stores it in top-level event fields for
 easier analysis in Kibana.
+<6> Local lookup queries could be use also prepared statements.
```

Comment:
"Local lookup queries could also use prepared statements."
... but @karenzone should double check on us :wink:

Reply:
I like it more, thanks